beta/experience-platform-extension/tutorials/README.md

```diff
@@ -64,3 +64,20 @@ Get started with [Assignment 3 - XDM implementation](https://aep-sdks.gitbook.io
 * [ ] What type of visitor identifiers do you currently sync using the AEP SDK?
 * [ ] Do you prefer syncing the visitor identifiers using a specialized API (example [syncIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#syncidentifier)), XDM format, remote rules or a combination of these?
 
+### Assignment 4 (week Nov 10)
+
+In this assignment you will learn how to:
+
+1. Update an existing XDM Schema
+2. Create data elements and rules for mobile properties in Adobe Experience Launch
+3. Attach new data to XDM events
+4. Validate the rules are executed correctly with Project Griffon
```

Comment:
"... executed correctly with Assurance (formerly Project Griffon)"

Reply:
Updated to `AEP Assurance (Project Griffon)`, we use both the names in our docs for now.